pkgs/development/python-modules/typedload/default.nix

```diff
@@ -0,0 +1,43 @@
+{
+  attrs,
+  buildPythonPackage,
+  fetchPypi,
+  lib,
+  pytestCheckHook,
+  python,
+  setuptools,
+}:
+
+let
+  typedload = buildPythonPackage rec {
+    pname = "typedload";
+    version = "2.37";
+    format = "pyproject";
+
+    src = fetchPypi {
```

Comment:
Recently a call has been made to prefer fetching python modules from source if possible. Can you look into this?

Reply:
Changed to use `fetchFromGitea` and point to codeberg in 0f9707085d655e1fc51539fb1f0264e5ac48767d; thanks!